docker/maistra-builder_1.2.Dockerfile

```diff
@@ -48,6 +48,9 @@ RUN curl -sfL https://install.goreleaser.com/github.com/ValeLint/vale.sh -o ./va
     chmod +x ./vale.sh && ./vale.sh -b /usr/local/bin ${VALE_VERSION} && \
     rm -f ./vale.sh
 
+# grafana
+RUN dnf install -y npm
```

Comment:
Please put this along with the other DNF calls on the top. Grouping all DNF calls helps to keep the image size smaller.

Reply:
Good point. Done